index.html

```diff
@@ -884,6 +1017,18 @@ <h3>
           <li>Return |buttons|.
           </li>
         </ol>
+        <p>
+          To <dfn data-lt="initializing touches">initialize touches</dfn> for a
```

Comment:
We problem don't need this initializer anymore, as the initial sequence is empty.